RunTests.ps1

```diff
@@ -0,0 +1,77 @@
+
+$DnxRuntimes = @(
+	'dnx-clr-win-x86.1.0.0-beta4', 
+	'dnx-coreclr-win-x86.1.0.0-beta4');
```

Comment:
will CI work once we will move to beta5? Is there a way to avoid having constants here? Perhaps getting it from global.json or something.